.github/workflows/on-push-to-branch.yml

```
@@ -58,3 +58,96 @@ jobs:
       microsoft_graph_client_secret: ${{ secrets.MICROSOFT_GRAPH_CLIENT_SECRET }}
       microsoft_graph_b2c_tenant_id: ${{ secrets.MICROSOFT_GRAPH_B2C_TENANT_ID }}
       microsoft_graph_b2c_tenant_name: ${{ secrets.MICROSOFT_GRAPH_B2C_TENANT_NAME }}
+
+  publish_webapp_image:
+    name: Publish Webapp
+    needs:
+      - test_backoffice
+      - test_service
+      - test_webapp
+    uses: ./.github/workflows/_reusable-publish-webapp-image.yml
+    with:
+      tag: ${{ github.sha }}
+      registry_url: 232705206979.dkr.ecr.eu-west-2.amazonaws.com
+      push_to_ecr: false
+    secrets:
+      aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

I wonder whether we only need the following, or can make it so that we only need it?
```suggestion
      tag: ${{ github.sha }}
      push_to_ecr: false
```
This would apply to the others if it's possible, but would be happy for that to be in a followup pull request.